src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,3 +303,93 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Footer`, () => {
+  test.beforeEach(async ({ dashboardPage, page }) => {
+    await dashboardPage.open();
+    try {
+      await checkAuthState(page);
+    } catch {
+      console.log("[E2E_LOG] - No fxa auth required, proceeding...");
+    }
+  });
+
+  test("Verify that the site footer is displayed correctly", async ({
+    dashboardPage,
+    page,
+    context,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463570",
+    });
+
+    const clickOnATagCheckDomain = async (
+      aTag: Locator,
+      expectedStrings: string[],
+    ): Promise<boolean> => {
+      const href = await aTag.getAttribute("href");
+      if (href === null) return false;
+
+      const target = await aTag.getAttribute("target");
+
+      if (target === "_blank") {
+        //opens link in a new tab
+        const newPage = await context.newPage();
+        await newPage.goto(href);
+        const currentUrl = newPage.url();
+        await newPage.close();
+        return expectedStrings.every((expectedString) =>
+          currentUrl.includes(expectedString),
+        );
+      } else {
+        //opens link in the same tab
+        await page.goto(href);
+        const currentUrl = page.url();
+        await page.goBack();
+        return expectedStrings.every((expectedString) =>
+          currentUrl.includes(expectedString),
+        );
+      }
+    };
+    await dashboardPage.goToDashboard();
+    expect(page.locator("footer a >> img")).toBeTruthy();
```

Comment:
We might also need to verify this.
This still seemed to pass locally if I created a simple test case but changed `>> img` to `>> missing`:
```js
import { test, expect } from '@playwright/test';

test('has img', async ({ page }) => {
  await page.goto('https://monitor.firefox.com');
  await expect(page.locator("footer a >> missing")).toBeTruthy(); // 1 passed (1.5s)
});
```

```sh
> jest-test@1.0.0 test
> playwright test tests


Running 1 test using 1 worker

  ✓  1 tests/monitor.spec.ts:3:5 › has title (823ms)

  1 passed (1.5s)
```

---

We could try asserting that the image has a specific `alt=""` attribute though. That seems to work locally, but fails after 30s if I pass an invalid img locator:
```js
import { test, expect } from '@playwright/test';

test('has img', async ({ page }) => {
  await page.goto('https://monitor.firefox.com');
  // Test timeout of 30000ms exceeded.
  // await expect(page.locator("footer a >> missing").getAttribute("alt")).resolves.toBe("Mozilla");

  // 1 passed (1.7s)
  await expect(page.locator("footer a >> img").getAttribute("alt")).resolves.toBe("Mozilla");
});
```

Reply:
I didn't realize that locator did not return null for an invalid selector, making .toBeTruthy to return true. Changing it to .toBeVisible indeed cause it to fail. Will incorporate that!